internal/datastore/crdb/migrations/driver.go

```diff
@@ -60,10 +62,24 @@ func (apd *CRDBDriver) Version(ctx context.Context) (string, error) {
 	return loaded, nil
 }
 
-// WriteVersion overwrites the value stored to track the version of the
-// database schema.
-func (apd *CRDBDriver) WriteVersion(ctx context.Context, version, replaced string) error {
-	result, err := apd.db.Exec(ctx, queryWriteVersion, version, replaced)
+// Conn returns the underlying pgx.Conn instance for this driver
+func (apd *CRDBDriver) Conn() *pgx.Conn {
+	return apd.db
+}
+
+func (apd *CRDBDriver) RunTx(ctx context.Context, f migrate.TxMigrationFunc[pgx.Tx]) error {
```

Comment:
Add doc comments for all exported functions

Reply:
This is not required by the linter, and is actually noisy if you're implementing an interface.